src/lib/components/CancelModal/CancelModal.js

```diff
@@ -53,13 +53,20 @@ export default class CancelModal extends Component {
   };
 
   render() {
-    const { buttonText, cancelText, content, header } = this.props;
+    const { buttonText, cancelText, content, header, isLoading } = this.props;
     const { open, value, showPopup } = this.state;
     return (
       <Modal
         basic
         size="small"
-        trigger={<Button primary content={buttonText} onClick={this.show} />}
+        trigger={
+          <Button
+            primary
+            content={buttonText}
+            onClick={this.show}
+            loading={isLoading}
```

Comment:
Could you please:
```suggestion
            loading={isLoading}
            disabled={isLoading}
```
I think we should add to all buttons with a loading state: my fear (did not test it!) is that you can still click while loading? We should maybe test it...

Reply:
yes, true, good point